docs/en/integrations/language-clients/java/index.md

```diff
@@ -281,6 +280,34 @@ try (PreparedStatement ps = conn.prepareStatement("insert into mytable values(tr
 ```
 
 ### Advanced API
+
+#### Connect to ClickHouse with SSL
+
+To establish a secure JDBC connection to ClickHouse using SSL, you'll need to configure your JDBC properties to include the SSL parameters. This typically involves specifying the SSL properties such as sslmode and sslrootcert in your JDBC URL/Properties object.
+
+These properties ensure that your Java application communicates with the ClickHouse server over an encrypted connection, enhancing data security during transmission.
+
+```java
+  String url = "jdbc:ch://your-server:8443/system";
+         
+  Properties properties = new Properties();
+  properties.setProperty("ssl", "true");
+  properties.setProperty("sslmode", "strict"); // NONE to trust all servers; STRICT for trusted only
```

Comment:
Maybe it would be easier to digest if we moved the property description outside of the code block? for example:
```
Properties:
- sslmode. valid values: 
    - 'strict' - ...
    - 'none' - ...
```

Reply:
Added. Could you please verify?